crates/metaslang/graph_builder/src/stack_graph/functions.rs

```diff
@@ -0,0 +1,163 @@
+// -*- coding: utf-8 -*-
+// ------------------------------------------------------------------------------------------------
+// Copyright © 2021, stack-graphs authors.
+// Licensed under either of Apache License, Version 2.0, or MIT license, at your option.
+// Please see the LICENSE-APACHE or LICENSE-MIT files in this distribution for license details.
+// ------------------------------------------------------------------------------------------------
+
+//! Define metaslang_graph_builder functions
+
+use metaslang_cst::KindTypes;
+
+use crate::functions::Functions;
+
+pub fn default_functions<KT: KindTypes + 'static>() -> Functions<KT> {
+    let mut functions = Functions::stdlib();
+    path::add_path_functions(&mut functions);
+    functions
+}
+
+pub mod path {
+    use std::path::{Component, Path, PathBuf};
+
+    use metaslang_cst::KindTypes;
+
+    use crate::execution::error::ExecutionError;
+    use crate::functions::{Function, Functions, Parameters};
+    use crate::graph::{Graph, Value};
+
+    pub fn add_path_functions<KT: KindTypes + 'static>(functions: &mut Functions<KT>) {
```

Comment:
how are these path functions used? is it to resolve imports?
My understanding is that we will resolve all import paths using an external service, not from `msgb` files. If so, are these still needed?

Reply:
Yes, these were again imported from the `tree-sitter-stack-graph` crate, but are not useful for us. I will remove them.